### PR TITLE
new dashboard

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9881

## Description
Added new dashboard that created by Joe for XSOAR insights.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by karinafishman** [comment]
https://github.com/demisto/content/pull/33726
PR for the script

File: Packs/CommonDashboards/ReleaseNotes/1_4_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### New: XSOAR Automation Insights
+
+- Added new Dashboard for XSOAR Automation Insights.
```
Comment (ssokolovich) [inline Packs/CommonDashboards/ReleaseNotes/1_4_1.md:6]:
```suggestion
- Added a new Dashboard for XSOAR Automation Insights.
```

**Review by ssokolovich — APPROVED** [review]

File: Packs/CommonDashboards/ReleaseNotes/1_4_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### New: XSOAR Automation Insights
+
+- Added a new Dashboard for XSOAR Automation Insights.
```
Comment (ShirleyDenkberg) [inline Packs/CommonDashboards/ReleaseNotes/1_4_1.md:6]:
```suggestion
Added a new Dashboard for XSOAR Automation Insights.
```

**Comment by ShirleyDenkberg** [comment]
@ssokolovich Doc review completed.

**Pull request merged**